capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -1,23 +1,72 @@
 import React, { Component } from 'react';
 
+import BookSearchTile from './BookSearchTile';
+import AssignmentCard from './AssignmentCard';
+ 
+import '../styles/Groups.css';
+
 
 class ClubPage extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      club: {}
+      club: {},
+      book: {},
+      assignments: [],
+      owner: {}, 
+      members: [], 
+      bookLists: []
     }
   }
 
-  fetchClubData = () => {
-    fetch(`/api/clubs?id=${this.props.match.params.id}`)
-        .then(response => response.json()).then(res => this.setState({club: res[0]}))
+  fetchData = async () => {
+    await fetch(`/api/clubs?id=${this.props.id}`)
+        .then(response => response.json()).then(response => this.setState({club: response[0]}))
```

Comment:
I had Geoff comment this on one of my PRs - maybe you could change `response` in your second `then` to be `clubJson` instead since it is no longer just the response from the fetch, but the jsonified version?

ex: `.then(clubJson => this.setState({club: clubJson[0]}))`

Same applies to your other fetches

Reply:
Good idea, I'll change this for readability.